config.toml

```diff
@@ -1,29 +1,31 @@
 title = "Harbor"
 baseURL = "https://goharbor.io"
 disableKinds = ["taxonomy", "taxonomyTerm"]
-copyright = "The Harbor authors"
+copyright = "Harbor authors"
 
 [params]
 tagline = "Manage and serve container images in a secure environment"
 description = "Our mission is to be the most secure, performant, scalable, and available cloud native repository for Kubernetes"
 intro = "Want to know more about what Harbor is and how it works? Watch this intro video to see how Harbor is helping organizations deliver a consistent image management solution across a variety of cloud native compute platforms like [Kubernetes](https://kubernetes.io) and [Docker](https://docker.com)."
 font_awesome_version = "5.12.0"
-favicon = "favicon.png"
+favicon = "favicon.svg"
 cncf_status = "incubating"
 repo = "goharbor/harbor"
 twitter = "project_harbor"
 slack = "https://cloud-native.slack.com/messages/harbor"
+github = "https://github.com/goharbor/harbor"
 user_group = "https://lists.cncf.io/g/harbor-users"
 dev_group = "https://lists.cncf.io/g/harbor-dev"
+logo_repo = "https://github.com/cncf/artwork/tree/master/projects/harbor"
```

Comment:
I think it'd be better to point users to this URL: https://branding.cncf.io/projects/harbor.

That page makes it easier to download the images and also has information about the project's color scheme.

Reply:
Definitely, will change